### PR TITLE
Group LFB data by property type

### PR DESCRIPTION
**Pull request by Eleonore9**
* Add cleaned LFB historical data
* Workflowfn to group LFB historical data on property type
* Return number of fires, average and standard deviation of the number of pumps attending
* Rename the function from "filter" to "group" because it's more accurate

**Note**: I commented out the acceptance test because it will fails at that stage.

### CODE REVIEW

File: data/fire_station_data.edn [hunk 1]
```diff
@@ -1 +1 @@
...
...andsworth", :area_km2 "16.991871", :radius "2.325657009", :lat "51.45577618", :long "-0.200077696"} {:station "Wimbledon", :area_km2 "15.432886", :radius "2.216402533", :lat "51.414799", :long "-0.196179825"} {:station "Tooting", :area_km2 "12.334748", :radius "1.981482332", :lat "51.43706304", :long "-0.16131744"} {:station "Mitcham", :area_km2 "14.077216", :radius "2.116817664", :lat "51.39664533", :long "-0.171420455"} {:station "Wallington", :area_km2 "19.842482", :radius "2.513176911", :lat "51.36162737", :long "-0.146844232"} {:station "Sutton", :area_km2 "20.019023", :radius "2.524332176", :lat "51.36828919", :long "-0.209746011"} {:station "Surbiton", :area_km2 "20.773406", :radius "2.571454938", :lat "51.39150971", :long "-0.296664384"} {:station "New Malden", :area_km2 "18.152957", :radius "2.403802337", :lat "51.39834884", :long "-0.243980855"} {:station "Kingston", :area_km2 "14.280161", :radius "2.132021675", :lat "51.42866546", :long "-0.305194454"} {:station "Richmond", :area_km2 "15.555469", :radius "2.225187535", :lat "51.46672988", :long "-0.283455013"} {:station "Twickenham", :area_km2 "27.974534", :radius "2.984052736", :lat "51.43444023", :long "-0.346214694"}]
```
Comment (seb231) [inline data/fire_station_data.edn]: Ah! Should it be stored in a vector? my bad.

File: src/kixi/gwyn/schemas.clj [hunk 2]
```diff
@@ -38,7 +36,10 @@
   {})
 
 (def CommercialProperties
-  {})
+  (make-ordered-ds-schema [[:property-type s/Str]
+                           [:num-fires s/Int]
+                           [:avg-pumps-attending s/Num]
```
Comment (seb231) [inline src/kixi/gwyn/schemas.clj]: Could we make this a double?

File: src/kixi/gwyn/schemas.clj [hunk 3]
```diff
@@ -38,7 +36,10 @@
...
+                           [:sd-pumps-attending s/Num]]))
```
Comment (seb231) [inline src/kixi/gwyn/schemas.clj]: and this to be safe

File: data/fire_station_data.edn (hunk 1 above)
Comment (Eleonore9) [inline data/fire_station_data.edn]: I thought it'd be easier when reading it to convert it into a dataset if it's a vector of maps.

File: src/kixi/gwyn/schemas.clj (hunk 3 above)
Comment (Eleonore9) [inline src/kixi/gwyn/schemas.clj]: Sure. I remember it failed before, so I'm going to investigate that!

File: test/kixi/gwyn/gwyn_test.clj [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+(def data-info
+  {:lfb-historic-incidents ["data/lfb_historical_fire_non_residential.csv"
+                            sc/LfbHistoricIncidents]})
+
+(def test-data
+  (reduce merge (map (fn [[k [p s]]]
+                       (hash-map k (tu/csv-to-dataset p s)))
+                     data-info)))
+
+(deftest filter-by-commercial-properties-type-test
```
Comment (Eleonore9) [inline test/kixi/gwyn/gwyn_test.clj]: Sure!

**Pull request merged**